docs/antrea-network-policy.md

```diff
@@ -1062,6 +1065,37 @@ spec:
       - fqdn: "svcA.default.svc.cluster.local"
 ```
 
+## Node Selector 
+
+NodeSelector selects certain Nodes which match the label selector. Add Node IPs to address group memberSet.
```

Comment:
```suggestion
NodeSelector selects certain Nodes which match the label selector. It adds Node IPs to address group memberSet.
```
And this is user facing doc, I'm not sure if users can understand "address group memberSet". At least there is no other fields mentioning it. Could you aligh with the description of other similar fields?

Reply:
Updated.